docs/schema.json

```diff
@@ -344,7 +356,6 @@
           "name": "my-channel2"
         }
       ],
-      "additionalItems": true,
```

Comment:
it should probably be reverted

Reply:
nope, it should not https://json-schema.org/understanding-json-schema/reference/array.html#list-validation